### ISSUE TITLE
Frontend Team Meeting : Planning Meeting Before Final Milestone

### ISSUE THREAD

**Issue by yigitsarioglu** [bounswe/2021SpringGroup4#238]
A meeting will be held on 26 December 2021 (26/12/2021) at 21:00.
Participants : Frontend team ( Yiğit Sarıoğlu, Yağmur Selek, Ali Alperen Durak)
Backend Team (Tolga Kerimoğlu)

Our task: Final milestone will be held next week. Preliminary preparations and plans are needed for this.

Topics to be discussed: 

- Comment Functionality
- Equipment Post & Equipment Cards & Equipment Search
- Event Page Completion & Participants 
- Badge Implementation

**Comment by yagmurse** [comment]
We held the meeting about 5 hours long. During the meeting we evaluated the completed and in progress tasks written in the description (Topics to be discussed). Everyone in frontend team and Tolga from the android team joined the meeting. We did peer writing for some bugs and manually merged the codes. We went over our scenario and did some tests.  Everything has been done can be found [under this commit](https://github.com/bounswe/2021SpringGroup4/commit/38277eccdc80d05ba747262ce63f28a31b0d7a64)
